src/library/libraryview.h

```diff
@@ -14,6 +14,7 @@ class LibraryView {
     virtual ~LibraryView() {};
 
     virtual void onShow() = 0;
+    virtual bool hasFocus() = 0;
```

Comment:
I was originally trying to do what you said, but I don't really know how to do that in C++ (I've only just started learning). LibraryView is an interface that doesn't have knowledge of whether or not the classes that implement it are QWidgets or not so we can't add it here. Where would I put the default implementation?